sdk/storage/storage-blob/review/storage-blob.api.md

```diff
@@ -38,10 +38,10 @@ export interface AccessPolicy {
 }
 
 // @public
-export type AccessTier = 'P4' | 'P6' | 'P10' | 'P15' | 'P20' | 'P30' | 'P40' | 'P50' | 'P60' | 'P70' | 'P80' | 'Hot' | 'Cool' | 'Archive';
+export type AccessTier = string;
```

Comment:
We need to doc the supported strings or a link to an official doc for this somewhere.
And it's better to be done in the common swagger so all languages can benefit.

Reply:
Updated this to be the same definition with the one generated with V5 generator.